ios/Classes/SingleFactorAuthFlutterPlugin.swift

```diff
@@ -46,7 +46,7 @@ public class SingleFactorAuthFlutterPlugin: NSObject, FlutterPlugin {
             case "init":
                 let args = call.arguments as? String
                 guard let data = args?.data(using: .utf8) else {
-                    return result(throwKeyNotGeneratedError())
+                    return result(throwParamMissingError(param: args!))
```

Comment:
Forced unwrap of args here could fail if args is null, or can args never be null?

Reply:
Please check other places for this as well